opentripplanner-geocoder/src/main/resources/org/opentripplanner/geocoder/application-context.xml

```diff
@@ -410,6 +410,13 @@
 		<property name="resultLimit" value="50" />
 	</bean>
 
+	<bean id="geocoderPelias" class="org.opentripplanner.geocoder.pelias.PeliasGeocoder">
+                <property name="apiKey" value="search-hAyPn56" />		

```

Comment:
From https://github.com/CUTR-at-USF/usf-mobullity/issues/176#issuecomment-242398716, I believe this should be:

    value="search-zso6JBE"